spring-integration-core/src/main/java/org/springframework/integration/util/MessagingMethodInvokerHelper.java

```diff
@@ -545,6 +564,11 @@ else if (!Modifier.isPublic(method1.getModifiers())) {
 				}
 				return;
 			}
+			if (AnnotationUtils.getAnnotation(method1, Default.class) != null) {
```

Comment:
Well, my idea to always have it explicitly. If you think the method must be default, so make it like that with declaring `@Default` exactly on place!